src/corppa/poetry_detection/passim/get_passim_excerpts.py

```diff
@@ -0,0 +1,82 @@
+"""
+Gather excerpts from passim page-level results.
```

Comment:
why make this separate from the other script for processing passim results?

Reply:
Because it was a secondary thing. We need page-level results full-stop, but it's nice to have the excerpts in a tsv/csv format so I can copy paste into a worksheet for manual stuff.

Fundamentally, some times we want to operate at the page-level and other times we want to operate at the excerpt-level, but not all pages even have excerpts. And we want to know about these cases (e.g., pages w/o any poems and pages for passim does not find any poesm).

Reply:
Another way to look at the outputs are for two separate things:

(1) Creating page-level evaluations.
(2) Identified poetry excerpts (for building the excerpt data set whose records fundamentally operate at the excerpt-level and not page-level).

Reply:
To better answer the question: there's no real reason why these are two separate scripts. They could probably be combined, but it's worth thinking what output we specifically want/need.